core/jvm/src/main/scala/cats/parse/BitSet.scala

```diff
@@ -64,7 +64,7 @@ object BitSetUtil {
 
   def bitSetForRange(count: Int): BitSet = {
     val bs = new BitSet(count)
-    bs.flip(0, count - 1)
```

Comment:
That was an actual source of the failed test. BitSet.flip upper bound seems to be exclusive despite being named  `toIndex`